src/world/Network/Handlers/PacketHandlers.cpp

```diff
@@ -239,73 +202,57 @@ void Sapphire::Network::GameConnection::updatePositionHandler( FrameworkPtr pFw,
   if( !player.hasInRangeActor() )
     return;
 
-  auto moveState = *reinterpret_cast< uint8_t* >( &copy.data[ 0x19 ] );
-  auto moveType = *reinterpret_cast< uint8_t* >( &copy.data[ 0x18 ] );
-
-  uint8_t unk1 = 0;
-  uint8_t unk2 = 0;
-  uint8_t unk3 = moveState;
-  uint16_t unk4 = 0;
+  auto clientAnimationType = updatePositionPacket.data().clientAnimationType;
+  auto animationState = updatePositionPacket.data().animationState;
+  auto animationType = updatePositionPacket.data().animationType;
+  auto headRotation = updatePositionPacket.data().headPosition;
+  uint8_t unkownRotation = 0;
```

Comment:
if we're not going to set this, just make it an optional param and ignore it in here

Reply:
its already optional and only set while falling

Reply:
ah missed that, np